proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,227 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **About**
```

Comment:
I think all contributing adjacent things, should be under another top-level group not "about".

For example:
```md
**Contributing to Kubeflow**
- Community
   - Events Calendar
   - Google Summer of Code (might be able to be removed)
- Documentation
   - Overview (currently called "docs")
   - Style Guide (currently called "Style Guide for the Kubeflow Docs")

```

Reply:
After taking a quick look at a few other open source docs sites, "About" is one of the headings that is often used for this sort of content. "Community" is another term that is used frequently.

Reply:
IMO "About" is quite a general term, forcing the user to click to see what is underneath. 

If we look at what's inside today, part could fit into "Community" (`About/Docs`, `About/Google Summer of Code`, ...) and the rest is "Getting started" related things (`About/Kubeflow`, `About/Use Cases`, ...)

Reply:
@thesuperzapper @joeliedtke I have refactored the "About" section according to this discussion. WDYT?

Reply:
Looks fine to me

Reply:
@RFMVasconcelos looks fine to me